src/Compilers/VisualBasic/Portable/Binding/ImportAliasesBinder.vb

```diff
@@ -22,8 +23,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
     ''' </summary>
     Friend Class ImportAliasesBinder
         Inherits Binder
+        Implements IImportChain
```

Comment:
in VB things are a bit different.  the binder is *itself* the link in the chain.  So two of the binders then expose the relevant information.  Specifically ImportAliasesBinder and TypesOfImportedNamespacesMembersBinder.  I got thes two binders from CreateBinderForSourceFile which showed me how you guys stitch up that chain yourself.

Reply:
not sure how acceptable/appropriate this approach is for you two.